### ISSUE TITLE
error bars in ezPlot

### ISSUE THREAD

**Issue by dfeistauer** [mike-lawrence/ez#61]
Hello Mike,

I would like to know why do you calculate for each group the same length of error bars? Would it not make more sense to use the SE of each group?
	
if(is.null(bar_size)){
		bar_size = stats$FLSD
	}
	stats$lo = stats$Mean-bar_size/2
	stats$hi = stats$Mean+bar_size/2

I have an one-factor-Design with repeated measures.
![rplot](https://user-images.githubusercontent.com/44641119/48788427-1a72c900-eceb-11e8-854b-fee56c098247.jpeg)

Greetings
Daniela

**Comment by mike-lawrence** [comment]
Different error bars answer different questions. An error bar reflecting the standard error answers the question "if the observed mean was the true mean, the observed magnitude of noise was the true magnitude of noise, and I ran many experiments similar to mine, what is the range in which most means from these new experiments would fall?". However, I've found that most people *interpret* an error bar as a method of comparison (*especially* for within-subject designs), a question the standard error is not well-suited to address directly, so instead ezPlot uses Fisher's Least Significant Difference, which is explicitly designed for comparisons.
